make run-local is broken

I tried to follow https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md#run-locally-with-minikube and got:
```
$ make run-local
./scripts/package-release.sh  deploy/tectonic-alm-operator/manifests/ deploy/tectonic-alm-operator/values.yaml
Usage: ./scripts/package-release.sh semver chart values
* semver: semver-formatted version for this package
* chart: the directory to output the chart
* values: the values file
make: *** [tectonic-release] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -63,11 +63,13 @@ manifests: $(CHARTS) build/chart/Chart.yaml build/chart/values.yaml
 	helm template -n olm -f build/chart/values.yaml build/chart --output-dir build/resources
 
 run-local: values_file = Documentation/install/local-values.yaml
+run-local: ver=0.0.0-local
 run-local: release
 	. ./scripts/build_local.sh
 	. ./scripts/install_local.sh local build/resources
 
 run-local-shift: values_file = Documentation/install/local-values-shift.yaml
+run-local-shift: ver=0.0.0-local
 run-local-shift: manifests
 	sed -i 's/rbac.authorization.k8s.io/authorization.openshift.io/' build/resources/02-alm-operator.rolebinding.yaml
 	. ./scripts/build_local_shift.sh

